README.md

```diff
@@ -27,7 +27,7 @@ Try out this simple youtube playlist.
 -   Step 1: Install git on your local machines
 -   Step 2: Create a GitHub account
 -   Step 3: Create a Pull request
--   Step 4: Add < Name - GitHub username > to [Members](https://github.com/opensourcely/git-sandbox/edit/master/README.md) along with the domain you are interested in.
+-   Step 4: Add < Name - GitHub username > to [Members](https://github.com/opensourcely/Getting-Started/edit/master/README.md) along with the domain you are interested in.
```

Comment:
1. The link should be pointing to the member.md file.
2. Should add an explicit  Step about forking the repo` ?
3. Should the steps 3 and 4 be interchanged ? (One creates a PR after editing the file, not before)

Reply:
1. Fixed with 3094160584973724c0934ca5d6e001f68469c9cb
2. Should add an explicit Step about forking the repo` ?
Since the user will not have write permission GitHub will automatically create fork for them, hence the user need not fork explicitly.
3. Yep.Fixed in 3094160584973724c0934ca5d6e001f68469c9cb